tests/e2e/e2e_common.go

```diff
@@ -94,8 +94,9 @@ const (
 	passorwdFilePath                           = "/etc/vmware/wcp/.storageUser"
 	podContainerCreatingState                  = "ContainerCreating"
 	poll                                       = 2 * time.Second
-	pollTimeout                                = 5 * time.Minute
+	pollTimeout                                = 10 * time.Minute
```

Comment:
Don't change this please, this is used in many places

Reply:
done